prov/rxm/src/rxm_ep.c

```diff
@@ -1032,6 +1032,9 @@ rxm_ep_postpone_send(struct rxm_ep *rxm_ep, struct rxm_conn *rxm_conn,
 	return FI_SUCCESS;
 }
 
+#define rxm_ep_tx_flags_inject(rxm_ep) \
```

Comment:
We can remove this macro if this is getting used only in `rxm_ep_inject_common`.

Reply:
We can set flags only to `FI_INJECT | flags` value